Assets/Unity Simple Liquid/Scripts/LiquidContainer.cs

```diff
@@ -338,6 +349,35 @@ private void OnDrawGizmosSelected()
         }
         #endregion
 
+        #region ChangeColor
+        private List<LiquidContainer> listLC = new List<LiquidContainer>();
+        public List<LiquidContainer> GetColorList
+        {
+            get { return listLC; }
+        }
```

Comment:
Why do you use list? Why not directly change color from SendLiquidContainer?

Reply:
yes, I agree, it is wiser to do as you suggest, this is genius